migrate/20230720_add_storage_cap.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:vm_host) do
+      add_column :total_storage_gib, Integer
+      add_column :available_storage_gib, Integer
```

Comment:
We have `used_cores/total_cores` and `used_hugepages_1g/total_hugepages_1g`. Why do we use `available_storage_gib` instead of `used_storage_gib`?

Reply:
Because `used_storage_gib` will be misleading, as we will have some storage space that's not used, but is not available either, but reserved for future.